python/tests/aggregate_tests/test_decimal_stddev_pop.py

```diff
@@ -5,7 +5,7 @@
 class aggtst_decimal_stddev_pop(TstView):
     def __init__(self):
         # Validated on Postgres
-        self.data = [{'c1': Decimal('2154.11'), 'c2': Decimal('2318.75')}]
+        self.data = [{"c1": Decimal("2154.11"), "c2": Decimal("2318.75")}]
```

Comment:
these should not be necessary I would think, the pre-commit should have fixed them
but maybe it didn't?

Reply:
This _is_ pre-commit :)